README.md

```diff
@@ -18,3 +18,6 @@ Visit https://shubham76.github.io/First-Pull-Request/ to see the web page.
 [github-user-name](github-profile-url)
```

Comment:
You must update your name and profile URL here. Along with a small message below.

Reply:
GARVIT CHHABRA     https://github.com/Garvit291/
i wanted to know github more

Reply:
Hi! Glad to help!
Just update your code as suggested below and raise the PR again.